.renovaterc.json

```diff
@@ -4,39 +4,46 @@
     ":enablePreCommit",
     "schedule:weekly"
   ],
-  "ignoreDeps": ["relekang/python-semantic-release"],
-  "pip_requirements": {
-    "fileMatch": ["^requirements(-[\\w]*)?\\.txt$"]
-  },
+  "enabledManagers": [
+    "pip_requirements",
+    "pre-commit"
+  ],
+  "ignoreDeps": [
+    "relekang/python-semantic-release"
+  ],
```

Comment:
I think we can also delete this, we had it because semantic-release was buggy for a while affecting our releases, but it's been running stable for a few months now after manually bumping it:

```suggestion
```

Reply:
Sure. Removing.